### PR TITLE
Added rancher2_cluster_sync resource

### PR DESCRIPTION
**Pull request by rawmind0**
`rancher2_cluster_sync` resource wait until a cluster is up and running in `active` state.

Fix issue #127

### CODE REVIEW

File: website/docs/r/clusterSync.html.markdown [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+sidebar_current: "docs-rancher2-resource-cluster_sync"
+description: |-
+  Provides a Rancher v2 Cluster Sync dummy resource. This can be used to create a Cluster Sync to wait for a rancher v2 Cluster resource `active` state.
+---
+
+# rancher2\_cluster\_sync
+
+Provides a Rancher v2 Cluster Sync dummy resource. This can be used to create a Cluster Sync to wait for a rancher v2 Cluster resource `active` state.
+
+This dummy resource doesn't create anything at rancher side. It's indeed to sync terraform resources that depends of rancher v2 Cluster resource in `active` state. This resource will wait until `cluster_id` is `active` on `terraform apply`. It also helps to sync `terraform destroy` dependencies, specially useful if cluster is using node pools.
```
Comment (cloudnautique) [inline website/docs/r/clusterSync.html.markdown]: change to `...anything on the Rancher side.`

Comment (cloudnautique) [reply website/docs/r/clusterSync.html.markdown]: Capitalize Rancher in the docs. 

It's used.. instead of It's indeed.

Comment (rawmind0) [reply website/docs/r/clusterSync.html.markdown]: Updated.

**Review by drpebcak — APPROVED** [review]
lgtm

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**